iceberg/iceberg-handler/src/test/queries/positive/fastforward_iceberg_branch.q

```diff
@@ -0,0 +1,23 @@
+-- SORT_QUERY_RESULTS
+set hive.explain.user=false;
+set hive.fetch.task.conversion=more;
+
+create external table ice01(a int, b string, c int) stored by iceberg stored as orc tblproperties ('format-version'='2');
+
+insert into ice01 values (1, 'one', 50), (2, 'two', 51), (111, 'one', 55);
+
+select * from ice01;
+
+-- create a branch named test1
+alter table ice01 create branch test1;
+
+select * from default.ice01.branch_test1;
+
+-- insert into main branch
+insert into ice01 values (10, 'ten', 53), (11, 'eleven', 52), (12, 'twelve', 56);
+select * from ice01;
+
+-- fastforward the branch
+explain alter table ice01 execute fastforward 'test1' to 'main';
```

Comment:
AFAIK you can only fast-forward to the main, isn't it? 
check this: https://github.com/trinodb/trino/issues/12844#issuecomment-1158294547

Reply:
hm, in docs you can provide diff target branch, so i guess we can ignore that point